dll/win32/shell32/wine/shellord.c

```diff
@@ -1388,8 +1388,15 @@ HRESULT WINAPI SHWinHelp(HWND hwnd, LPCWSTR pszHelp, UINT uCommand, ULONG_PTR dw
  */
 BOOL WINAPI SHRunControlPanel (LPCWSTR commandLine, HWND parent)
 {
+#ifdef __REACTOS__
+    HINSTANCE hr;
+    TRACE("SHRunControlPanel(%s, %p)n", debugstr_w(commandLine), parent);
```

Comment:
//TODO: Run in-process when possible, (HKLM\Software\Microsoft\Windows\CurrentVersion\Explorer\ControlPanel\InProcCPLs), and possibly some extra rules,

Reply:
Good point. I keep it for a future improvement.

Reply:
Comment added to the code.